Сетка личного проекта на флексах

AdamMalec opened the pull request:
---
:mortar_board: [Сетка личного проекта на флексах](https://up.htmlacademy.ru/htmlcss/20/user/276647/tasks/42)

:boom: https://htmlacademy-htmlcss.github.io/276647-device/7/

katemikulyanec commented:
Привет!)
Пару изменений
По главной
1) Перепроверь,пожалуйста, все паддинги/марджины ,добавь шде не хватает, удали лишнее
2) Здесь надо перепроверить оступы между элементами http://joxi.ru/KAgo53Kf4dJa0A
3)Этим элементам давно надо добавить hover, active, и картинки, а текст выровнять http://joxi.ru/LmGegNVte0L18A
4) Эти элементы пора сделать по пакету, уменьшить отступы и выровнять их http://joxi.ru/YmEag7VS0yWxXm
5) А этим секциям надо добавить паддинги и картинки http://joxi.ru/E2p138vC9N1yEA
6) В футере лучше удалить див обертку с классом footer-wrapper
Стили этой обертки задать самому тегу footer и внутри уже добавить обертку .container
7) Вот эти две сторочки лишние оно и так будет работать правильно
.main-footer {
    /* display: flex; */
    /* flex-direction: column; */
}
8) Здесь можно добавить 
 .footer-top {
    justify-content: space-between;
}
И обертку этим двум элементам http://joxi.ru/krDlgypsEj6X8r
А марждини им  потом уменьшить в соответствии с макетом
9) Тут тоже дисплей флекс удалить .footer-user
10) И добавь, пожалуйста,желтую полосочку в футере
11) Эту форму уже тоже пора привести в соответствие с макетом http://joxi.ru/Dr8yaLKC48E6Wm
12) Этим элементам надо задать ширину, чтобы потом ничего не вылазило (потому что картинки могут разной величины потом добавлять или текста добавят) и еще добавить этим li марждины снизу http://joxi.ru/LmGegNVte0Lz8A
13) А здесь надо явно задать ширину, чтобы цена не спрыгивала на следующую строчку http://joxi.ru/YmEag7VS0yWoXm

AdamMalec commented:
1. padding поубирал у <h> и <p> лишние. Где они там лишние, а где не хватает... хотел бы я сам знать. Буду доверстывать походу дела, буду разбираться.
2. Там я просто ничего не делал еще) Общая же сетка, а не микро. Пощади!
3. Катя, давай с этой секцией я начну разбираться со следующего "учебного блока"? Потому что для меня эта секция бесящая. И вообще, со следующей проверки пойду верстать от хедера до футера, все согласно макету. И тогда уже конкретный момент/вопрос будем разбирать по косточкам.
4. см. пункт 3
5. FIXED
6. FIXED
7. FIXED
8. Здесь он как бы сейчас не нужен. Но задумка такая, что после авторизации (Валентин Петухов) нужен.
9. Я задал явно через высоту и ширину для div с полосочкой. не стал заморачиваться с псевдо. если критично - позже исправим.
10. Здесь такая же тема. Стилизацию формы взяли лишь на лекции следующей за сетками, поэтому я не трогал здесь ничего от греха подальше чтобы не намудрить
11. FIXED
12. FIXED
